packages/logs/webpack.config.js

```diff
@@ -2,18 +2,10 @@ const path = require('path')
 
 const webpackBase = require('../../webpack.base')
 
-const datacenter = process.env.TARGET_DATACENTER || 'us'
-const withSuffix = process.env.WITH_SUFFIX || false
+const entry = path.resolve(__dirname, 'src/boot/logs.entry.ts')
 
-const suffix = withSuffix ? `-${datacenter}` : ''
-
-module.exports = (env, argv) => ({
-  entry: {
-    logs: path.resolve(__dirname, 'src/boot/logs.entry.ts'),
-  },
-  ...webpackBase(argv.mode),
-  output: {
-    filename: `datadog-logs${suffix}.js`,
-    path: path.resolve(__dirname, 'bundle'),
-  },
-})
+module.exports = (env, argv) => [
+  webpackBase({ mode: argv.mode, entry, datacenter: 'us', filename: 'datadog-logs.js' }),
+  webpackBase({ mode: argv.mode, entry, datacenter: 'us', filename: 'datadog-logs-us.js' }),
+  webpackBase({ mode: argv.mode, entry, datacenter: 'eu', filename: 'datadog-logs-eu.js' }),
```

Comment:
Since us/eu files are now built every time, they will end up in rum and logs npm packages.
I think it could be nice to not build them in this case or at least exclude them from the npm package.